Calibration/IsolatedParticles/plugins/IsolatedTracksNxN.cc

```diff
@@ -2760,18 +2761,24 @@ void IsolatedTracksNxN::printTrack(const reco::Track *pTrack) {
   if (printTrkHitPattern_) {
     const reco::HitPattern &p = pTrack->hitPattern();
 
-    edm::LogVerbatim("IsoTrack") << "default ";
+    std::ostringstream st1;
+    st1 << "default ";
```

Comment:
Why do you need this change? Is it that `LogVerbatim` didnt print out? You could use `LogPrint` instead